ibis/backends/dask/udf.py

```diff
@@ -165,9 +165,10 @@ def lazy_agg(*series: pandas.Series):
         return result
 
     @execute_node.register(
-        type(op), *(itertools.repeat(ddgb.SeriesGroupBy, nargs))
+        ops.ReductionVectorizedUDF,
+        *(itertools.repeat(ddgb.SeriesGroupBy, nargs)),
     )
-    def execute_udaf_node_groupby(op, *args, aggcontext, **kwargs):
+    def execute_udrf_node_groupby(op, *args, aggcontext, **kwargs):
```

Comment:
I think udaf is still the convention to call this (User defined aggregation functions), which is a common term in sql land.

Reply:
What would be a better term? I need to split out the handler for `ops.ReductionVectorizedUDF` and `ops.AnalyticVectorizedUDF`. I can switch it to `execute_reduction_node_groupby` and `execute_analytic_node_groupby`?

Reply:
`execute_reduction_node_groupby` and `execute_analytic_node_groupby` sounds good to me